build.gradle

```diff
@@ -202,6 +202,22 @@ model {
     }
 }
 
+project(':devApp') {
+    apply plugin: 'java'
+    apply plugin: 'application'
+
+    dependencies {
+        compile project(':')
+        runtime files(project(':').nativeTestFilesJar.archivePath)
+    }
+
+    sourceSets.main.java.srcDirs = [ "${rootDir}/src/dev/java" ]
```

Comment:
Why don't you just follow the gradle standard and create a new directory called `devApp` and put the source code in there?